platforms/documentation/docs/src/snippets/testing/jacoco-quickstart/kotlin/build.gradle.kts

```diff
@@ -85,7 +85,7 @@ tasks.test {
         isIncludeNoLocationClasses = false
         sessionId = "<auto-generated value>"
         isDumpOnExit = true
-        classDumpDir = null
+        classDumpDir = null as? File
```

Comment:
Shall we call `unset()` here?

Reply:
This actually doesn't unset the extension but just shows the default values